museum/sampleRecords/donors.destination.json

```diff
@@ -0,0 +1,578 @@
+      "ca_entities.DonorNotes":"Donor's father was assistant Engineer in Chief of WA.",
+      "ca_entities.LastEditBy":"IST",
+      "ca_entities.LastEditDate":"19 December 2000"
+    },
+    {
+      "entity_id":"8",
+      "id":"8",
+      "idno":"9",
+      "display_label":"ANDERSON Nell",
+      "ca_entities.idno":"9",
+      "ca_entities.preferred_labels.displayname":"ANDERSON Nell",
+      "ca_entities.preferred_labels.prefix":"Miss",
+      "ca_entities.preferred_labels.forename":"Nell",
+      "ca_entities.preferred_labels.middlename":"ND",
+      "ca_entities.preferred_labels.surname":"Anderson",
+      "ca_entities.AddressContainer.Address1":"Unit 1, 163 Royal Street",
+      "ca_entities.AddressContainer.TownCity":"YOKINE",
+      "ca_entities.AddressContainer.State":"WA",
+      "ca_entities.LastEditBy":"JILLM",
+      "ca_entities.LastEditDate":"16 March 2004"
+    },
+    {
+      "entity_id":"9",
+      "id":"9",
+      "idno":"10",
+      "display_label":"KIRWAN M",
+      "ca_entities.idno":"10",
+      "ca_entities.preferred_labels.displayname":"KIRWAN M",
+      "ca_entities.preferred_labels.prefix":"Miss",
+      "ca_entities.preferred_labels.forename":"M",
+      "ca_entities.preferred_labels.middlename":"M",
+      "ca_entities.preferred_labels.surname":"Kirwan",
+      "ca_entities.AddressContainer.Address1":"40 Webster Street",
+      "ca_entities.AddressContainer.TownCity":"NEDLANDS",
+      "ca_entities.AddressContainer.State":"WA",
+      "ca_entities.AddressContainer.PostCode":"6009",
+      "ca_entities.DonorNotes":"Address c\/o Mrs W. McManus",
+      "ca_entities.LastEditBy":"PENDS",
+      "ca_entities.LastEditDate":"13 November 2014"
+    },
+    {
+      "entity_id":"10",
+      "id":"10",
+      "idno":"11",
+      "display_label":"DAVEY A",
+      "ca_entities.idno":"11",
+      "ca_entities.preferred_labels.displayname":"DAVEY A",
+      "ca_entities.preferred_labels.prefix":"Mr",
+      "ca_entities.preferred_labels.forename":"A",
+      "ca_entities.preferred_labels.middlename":"A",
+      "ca_entities.preferred_labels.surname":"Davey",
+      "ca_entities.AddressContainer.Address1":"30 Simons Street",
+      "ca_entities.AddressContainer.TownCity":"COOLBELLUP",
+      "ca_entities.AddressContainer.State":"WA",
+      "ca_entities.AddressContainer.PostCode":"6163",
+      "ca_entities.LastEditBy":"ValH",
+      "ca_entities.LastEditDate":"18 March 2013"
+    },
+    {
+      "entity_id":"11",
+      "id":"11",
+      "idno":"12",
+      "display_label":"LLOYD, MRS Stephanie",
+      "ca_entities.idno":"12",
+      "ca_entities.preferred_labels.displayname":"LLOYD, MRS Stephanie",
+      "ca_entities.preferred_labels.prefix":"Mrs",
+      "ca_entities.preferred_labels.forename":"Stephanie",
+      "ca_entities.preferred_labels.middlename":"S.",
+      "ca_entities.preferred_labels.surname":"Lloyd, Mrs",
+      "ca_entities.AddressContainer.Address1":"1c Woodhouse Street, East Fremantle, 6158",
```

Comment:
This shouldn't have the suburb and postcode, but I can see that comes from the source data.  Something for the client to clean.